apps/studio/src/features/editing-experience/components/RootStateDrawer.tsx

```diff
@@ -177,31 +175,16 @@ export default function RootStateDrawer() {
         )}
       </VStack>
 
-      {hasChildpagesBlock && (
-        <BaseBlock
-          onClick={() => {
-            setDrawerState({ state: "childpagesEditor" })
-          }}
-          label={"Child pages"}
-          description={"Title, Description and Order of display"}
-          icon={BiListUl}
-          // NOTE: we have an inbuilt gap due to the outer `Stack`
-          // hence, we need a negative margin here
-          // so that it will fit the desired design
-          containerProps={{ mt: "-0.5rem" }}
-        />
-      )}
-
-      {(pageLayout !== "index" || savedPageState.content.length > 0) && (
-        <VStack w="100%" h="100%" gap="1rem">
-          {/* Custom Blocks Section */}
-          <Flex flexDirection="row" w="100%">
-            <VStack gap="0.25rem" align="start" flex={1}>
-              <Text textStyle="subhead-1">Custom blocks</Text>
-              <Text textStyle="caption-2" color="base.content.medium">
-                Use blocks to display your content in various ways
-              </Text>
-            </VStack>
+      <VStack w="100%" h="100%" gap="1rem">
+        {/* Custom Blocks Section */}
+        <Flex flexDirection="row" w="100%">
+          <VStack gap="0.25rem" align="start" flex={1}>
+            <Text textStyle="subhead-1">Custom blocks</Text>
+            <Text textStyle="caption-2" color="base.content.medium">
+              Use blocks to display your content in various ways
+            </Text>
+          </VStack>
+          {pageLayout !== ISOMER_USABLE_PAGE_LAYOUTS.Index && (
```

Comment:
question: do we need to also hide the "Add a new block" placeholder if the page has no existing blocks?